test/Writer/Extension/ITunes/EntryTest.php

```diff
@@ -158,8 +158,8 @@ public function testSetKeywordsThrowsExceptionIfMaxKeywordsExceeded()
         }, \E_USER_DEPRECATED);
         try {
             $entry->setItunesKeywords($words);
-            $this->fail('Expected exception when setting more keywords than allowed');
         } catch (ExceptionInterface $e) {
+            $this->assertInstanceOf(ExceptionInterface::class, $e);
```

Comment:
Use `expectException()` and omit catch block entirely, same for the rest.

Reply:
Please give me a hint how to restore the error handler then.

Reply:
@froschdesign I didn't say try-catch, only catch. It is optional in the presence of finally